validation/models.py

```diff
@@ -361,6 +362,8 @@ class Recording(models.Model):
     GOOD = "good"
     BAD = "bad"
     UNKNOWN = "unknown"
+    WRONG_SPEAKER = "wrong speaker"
```

Comment:
I just realized that the `quality` field should have the `choices` attribute :/

See: https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.Field.choices